pkg/api/customization/node/node.go

```diff
@@ -48,16 +50,30 @@ func Formatter(apiContext *types.APIContext, resource *types.RawResource) {
 	if nodeTemplateID == nil && customConfig == nil {
 		delete(resource.Links, "remove")
 	}
-	if resource.Values[client.NodeFieldWorker] != true ||
-		resource.Values["state"] == "registering" ||
-		resource.Values["state"] == "unavailable" {
+	if resource.Values[client.NodeFieldWorker] != true {
+		return
+	}
+	state := resource.Values["state"]
+	allowed := state == "active" || state == "cordoned" || state == "draining" || state == "drained"
```

Comment:
we can use the map for storing the states, and make a check to see if the key exists